Tests/conf.json

```diff
@@ -2,6 +2,11 @@
     "testTimeout": 160,
     "testInterval": 20,
     "tests": [
+	{
+            "integrations": "cyberint",
+            "playbookID": "playbook-Cyberint_Test",
+            "fromversion": "5.0.0"
+        },
```

Comment:
Please remove the change in this file. That file should not be a part of your PR.

Reply:
Are you sure? I couldnt' pass the CICD validation check without it.

Reply:
Yes, I am sure, if you cannot remove it I will do it after merge.